apps/nova/src/components/Layout/Sidebar.tsx

```diff
@@ -100,6 +100,10 @@ const Sidebar = () => {
 
   return (
     <div
+      style={{
+        boxShadow: "0px 14px 42px 0px rgba(0, 0, 0, 0.08)",
+        zIndex: 2,
+      }}
```

Comment:
No need to use css styles. you can do this with tailwind. Please define these values in tailwind config. `shadow-sidebar z-sidebar`